flow/packet-headers/snmpv2c.yaml

```diff
@@ -0,0 +1,366 @@
+          description: >-
+            A Sequence of variable_bindings.
+          type: array
+          items:
+            $ref: '#/components/schemas/Flow.Snmpv2c.VariableBinding'
+          x-field-uid: 4
+    Flow.Snmpv2c.BulkPDU:
+      description: The purpose of the GetBulkRequest-PDU is to request
+          the transfer of a potentially large amount of data, including, but
+          not limited to, the efficient and rapid retrieval of large tables.
+      type: object
+      properties:
+        request_id:
+          x-include: '#/components/schemas/Flow.Snmpv2c.Common/properties/request_id'
+          x-field-uid: 1
+        non_repeaters:
+          x-field-pattern:
+            description: >-
+                One variable binding in the Response-PDU is requested for the first non_repeaters variable bindings in the GetBulkRequest.
+            format: integer
+            length: 32
+            default: 0
+          x-field-uid: 2
+        max_repetitions:
+          x-field-pattern:
+            description: >-
+                A maximum of max_repetitions variable bindings are requested in the Response-PDU for each of the remaining variable bindings in the GetBulkRequest after the non_repeaters variable bindings.
+            format: integer
+            length: 32
+            default: 0
+            features: [count]
+          x-field-uid: 3
+        variable_bindings: 
+          description: >-
+            A Sequence of variable_bindings.
+          type: array
+          items:
+            $ref: '#/components/schemas/Flow.Snmpv2c.VariableBinding'
+          x-field-uid: 4
+    Flow.Snmpv2c.VariableBinding:
+      description: A Sequence of two fields, an object_identifier and the value for/from that object_identifier.
+      type: object
+      properties:
+        object_identifier:
+          x-include: '#/components/schemas/Flow.Snmpv2c.Common/properties/object_identifier'
+          x-field-uid: 1
+        value: 
+          $ref: '#/components/schemas/Flow.Snmpv2c.VariableBindingValue'
+          x-field-uid: 2
+    Flow.Snmpv2c.VariableBindingValue: 
+      description: >-
+          The value for the object_identifier as per RFC2578.
+      type: object
+      properties:
+        choice:
+          type: string
+          default: no_value
+          x-field-uid: 1
+          x-enum:
+            no_value:
+              x-field-uid: 1
+            integer_value:
+              x-field-uid: 2
+            string_value: 
+              x-field-uid: 3
+            object_identifier_value:
+              x-field-uid: 4
+            ip_address_value:
+              x-field-uid: 5
+            counter_value:
+              x-field-uid: 6
+            timeticks_value:
+              x-field-uid: 7
+            arbitrary_value:
+              x-field-uid: 8
+            big_counter_value:
+              x-field-uid: 9
+            unsigned_integer_value:
+              x-field-uid: 10
+        integer_value:
```

Comment:
We are proving OTG data type (e,g, `<int32>`, <uint32>`, `<string>` etc) within all SNMP property. But those data types are not one-to-one mapping with ASN.1 type. say 
- `community` and `error_status` are `string` but those are different ASN.1 type (which are `0x04` and `0x02`)
- Same also true for `request_id` and `integer_value`, both are `int32` with different ASN.1 type (`0x02` and `0x41`)

Yes, I agree this is not a goal for OTG model perspective. So, I am thinking it will better to mentioned what is the ANS.1 type for each field within description. It will reduced ramp-up and anomality at the time to develop SNMP protocol.